modular_zubbers/master_files/code/modules/loadout/loadout_preference.dm

```diff
@@ -0,0 +1,42 @@
+// Oh, I'm sorry, you were looking for GOOD code? Turn around and leave. - Rimi
+
+/datum/preference/loadout_index
+	savefile_key = "loadout_index"
+	savefile_identifier = PREFERENCE_CHARACTER
+	can_randomize = FALSE
+
+/datum/preference/loadout_index/apply_to_human(mob/living/carbon/human/target, value, datum/preferences/preferences)
+	return
+
+/datum/preference/loadout_index/create_informed_default_value(datum/preferences/preferences)
+	var/list/loadouts = preferences.read_preference(/datum/preference/loadout)
+	var/name
+	for (name in loadouts) // Lazy hack fraud
+		break
+	return name
+
+/datum/preference/loadout_index/deserialize(input, datum/preferences/preferences)
+	if (istext(input))
+		return input
+
+	var/list/loadouts = preferences.read_preference(/datum/preference/loadout)
+	var/name
+	for (name in loadouts) // Lazy hack fraud
+		break
```

Comment:
ditto, also since you use this exact same thing in the create_informed_default_value proc you can just avoid copypasta by calling it directly as a fallback